qiita_pet/webserver.py

```diff
@@ -106,6 +108,11 @@ def __init__(self):
             (r"/stats/", StatsHandler),
             (r"/download/(.*)", DownloadHandler),
             (r"/vamps/(.*)", VAMPSHandler),
+            # Plugin handlers - the order matters here so do not change
+            (r"/qiita_db/jobs/(.*)/heartbeat/", HeartbeatHandler),
```

Comment:
I thought your initial intent was to keep these handlers separate from the main server, just wanted to check with you.